README.md

```diff
@@ -2,73 +2,250 @@
+    // Prevents the default behavior on click and rather opens the suggestion
+    // in a new tab.
+    if (context.selectionMethod === "click") {
+      input.setVal("");
+
+      const windowReference = window.open(suggestion.url, "_blank");
+      windowReference.focus();
+    }
+  }
+});
+```
+
+Note that, by default, you can already open a new tab thanks to the CMD/CTRL + Click action.
+
+The method is called with the following arguments:
+
+- `input`: a reference to the search input element. It comes with the `.open()`, `.close()`, `.getVal()` and `.setVal()` methods.
+
+- `event`: the actual event triggering the selection.
+
+- `suggestion`: the object representing the current selection. It contains a `.url` key representing the destination.
+
+- `datasetNumber`: this should always be equal to 1 as `docs-searchbar.js` is searching into one dataset at a time. You can ignore this attribute.
+
+- `context`: additional information about the selection. Contains a `.selectionMethod` key that can be either `click`, `enterKey`, `tabKey` or `blur`, depending on how the suggestion was selected.
+
+##### `meilisearchOptions` <!-- omit in toc -->
+
+You can forward search parameters to the MeiliSearch API by using the `meilisearchOptions` key. Checkout out the [MeiliSearch documentation about search parameters](https://docs.meilisearch.com/guides/advanced_guides/search_parameters.html#search-parameters).
+
+For example, you might want to increase the number of results displayed in the dropdown:
+
+```javascript
+docsSearchBar({
+  meilsearchOptions: {
+    limit: 10
+  }
+});
+```
+
+##### More Examples <!-- omit in toc -->
+
+Here is a basic [HTML file](scripts/playground.html) used in the playground of this repository.
+
+As a more concrete example, you can [check out the configuration](https://github.com/meilisearch/vuepress-plugin-meilisearch/blob/bf2210e131eb5630f632d514f35710ee6576ec7a/MeiliSearchBox.vue#L64) applied in the MeiliSearch plugin for Vuepress.
+
+#### Styling <!-- omit in toc -->
+
+```css
+/* Main dropdown wrapper */
+.meilisearch-autocomplete .dsb-dropdown-menu {
+  width: 500px;
+}
+
+/* Main category */
+.meilisearch-autocomplete .docs-searchbar-suggestion--category-header {
+  color: darkgray;
+  border: 1px solid gray;
+}
+
+/* Category */
+.meilisearch-autocomplete .docs-searchbar-suggestion--subcategory-column {
+  color: gray;
+}
+
+/* Title */
+.meilisearch-autocomplete .docs-searchbar-suggestion--title {
+  font-weight: bold;
+  color: black;
+}
+
+/* Description */
+.meilisearch-autocomplete .docs-searchbar-suggestion--text {
+  font-size: 0.8rem;
+  color: gray;
+}
+
+/* Highlighted text */
+.meilisearch-autocomplete .docs-searchbar-suggestion--highlight {
+  color: blue;
+}
+```
+
+Here is the [CSS customization](https://github.com/meilisearch/vuepress-plugin-meilisearch/blob/bf2210e131eb5630f632d514f35710ee6576ec7a/MeiliSearchBox.vue#L77-L177) applied in the MeiliSearch plugin for Vuepress.
```

Comment:
What is the purpose of this information? To inform the user they can customize their CSS?

Reply:
Provide a concrete example with CSS customization for the search bar because it's impossible to detail all the possibilities in the README.

Reply:
@bidoubiwa do you think it needs to be more explicit?

Reply:
Oui, en effet c'est dans la section "more examples", ca me parait clair!

Reply:
In which case the "more examples" section should not be ##### and "styling" #### but the opposite !

Reply:
It's more The `More examples` section is for the first part of the customization (`Optional parameters`), not for the styling part.

But I can do a `More examples` section for the `Styling` section as well 🙂

Reply:
Ah oke! It's good as it is then no problem